components/ui/CardSubtitle.vue

```diff
@@ -0,0 +1,10 @@
+<script setup lang="ts">
+defineProps<{
+    forId: string;
+    text: string;
+}>();
+</script>
+
+<template>
+    <label :for="forId" class=" text-base font-medium leading-6 text-gray-900">{{ text }}</label>
+</template>
```

Comment:
pretty straight forward for me. dont create this component. 
> use label directly

Reply:
Already remove unnecessary components